yoyodyne/models/base.py

```diff
@@ -235,11 +235,33 @@ def validation_epoch_end(self, validation_step_outputs: Dict) -> Dict:
             self.log(metric, value, prog_bar=True)
         return metrics
 
+    def predict_step(
+        self,
+        batch: batches.PaddedBatch,
+        batch_idx: int,
+    ) -> torch.Tensor:
+        """Runs one predict step.
+
+        This is called by the PL Trainer.
+
+        Args:
+            batch (batches.PaddedBatch).
+            batch_idx (int).
+
+        Returns:
+            torch.Tensor: indices of the argmax at each timestep.
+        """
+        self.eval()
+        predictions = self(batch)
+        # -> B x seq_len x output_size.
```

Comment:
Are you sure this doesn't return `B x seq_len x 1`?

Reply:
Oh, mb.